tar.go

```diff
@@ -17,6 +17,9 @@ func init() {
 }
 
 type Tar struct {
+	// If true, leave only numeric user and group id
+	NumericUidGid bool
```

Comment:
```suggestion
	// If true, preserve only numeric user and group id
	NumericUIDGID bool
```

(Just a nit, don't mind me. I'll still merge this in a moment.)